src/main/java/com/iridium/iridiumteams/configs/Messages.java

```diff
@@ -169,6 +175,7 @@ public Messages(String team, String command, String pluginName, String color) {
         mustBeAPlayer = "%prefix% &7You must be a player to execute this command.";
         notAPlayer = "%prefix% &7That player doesn't exist.";
         unknownCommand = "%prefix% &7Unknown Command, Try /" + command + " help.";
+        topCommandHeader = "&8[" + color + "&l" + pluginName + "&r" + color + " Leaderboard - &l&a%sortType%&8]";
```

Comment:
I'd copy the Filler aswell from below, and then copy the same code from the help command

https://github.com/Iridium-Development/IridiumTeams/blob/master/src/main/java/com/iridium/iridiumteams/commands/HelpCommand.java#L67

This is used to center the message propperly on the chat window (for stuff like /is help and /is info, its especially helpful for /is info because the header includes the island name, so the length of the header changes alot which makes it hard to center propperly)